app/models/booking.rb

```diff
@@ -0,0 +1,8 @@
+class Booking < ActiveRecord::Base
+  belongs_to :event
+  belongs_to :applicant, foreign_key: 'applicant_id', class_name: 'User'
+
+  def formatted_booking_time
+    created_at.to_formatted_s(:long)
```

Comment:
Just checking `created_at` is intentionally the same as `booking_time`. The names make them sound like they serve different purposes.